src/View3d.ts

```diff
@@ -162,6 +166,10 @@ export class View3d {
     this.canvas3d.redraw();
   }
 
+  render(): void {
+    this.canvas3d.render();
+  }
```

Comment:
slightly hesitant to give clients this hook.  
Also, if you look at `redraw()`, the actual function responsible for a full redraw is called `onAnimationLoop`.  Make sure that you don't actually need the extra bits of code that happen in there.  

(Nit) I would maybe consider just passing a flag to the redraw call instead of adding this new function...

Reply:
That sounds reasonable to me! I'll make that change.